### PR TITLE
integration/cri-o: do not edit ctr.bats

### PR DESCRIPTION
**Pull request by littlejawa**
Make a copy of ctr.bats before editing it.
This prevents multiple lines from being added to the file in cri-o sources.
It also makes it possible to change the list of skipped tests and take the
change into account.

Fixes: #3622

Signed-off-by: Julien Ropé <jrope@redhat.com>

### CODE REVIEW

**Comment by littlejawa** [comment]
CC: @fgiudici

**Comment by fgiudici** [comment]
lgtm!

**Review by jodh-intel — APPROVED** [review]
Thanks @littlejawa.

lgtm

**Comment by fidencio** [comment]
@littlejawa, for completeness, shouldn't we also remove the file we created after the run?

**Comment by littlejawa** [comment]
@fidencio sure, it doesn't hurt. I'll add that.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @littlejawa!

File: integration/cri-o/cri-o.sh [hunk 1]
```diff
@@ -68,9 +68,10 @@ IFS=''
 
 # Skip CRI-O tests that currently are not working
 pushd "${crio_repository_path}/test/"
+cp $GOPATH/src/${crio_repository}/test/ctr.bats $GOPATH/src/${crio_repository}/test/ctr_kata_integration_tests.bats
```
Comment (uril) [inline integration/cri-o/cri-o.sh:71]: minor comment (it's good as is):
crio_repository_path is $GOPATH/src/${crio_repository} and the script just pushd into it
so you can simply cp ctr.bats ctr_kata_integration_tests.bats (no need for prefix)

Comment (littlejawa) [reply integration/cri-o/cri-o.sh]: Right. The "sed" line below was using the prefix, so I just copied it. But it's not needed.

**Comment by snir911** [comment]
some of the jobs seems stuck
/retest

**Pull request merged**